src/nora/components/KabobMenu/KabobMenu.js

```diff
@@ -19,24 +19,36 @@ const KabobIcon = () => {
   )
 }
 
-export const KabobMenu = ({ className, onClick, onKeyDown, children }) => {
-  const containerClasses = className
-    ? `${styles.Container} ${className}`
-    : styles.Container
-
+export const KabobMenu = ({
+  focusRef,
+  tabIndex,
+  onClick,
+  onKeyDown,
+  children,
+}) => {
+  const tabIndexResolved = tabIndex ? tabIndex : null
+  const focusRefResolved = focusRef ? focusRef : null
   return (
-    <div className={containerClasses} onKeyDown={onKeyDown}>
-      <button className={styles.Box} onClick={onClick}>
+    <>
+      <button
+        ref={focusRefResolved}
+        tabIndex={tabIndexResolved}
+        className={styles.Box}
```

Comment:
Was your thinking that we'd have an enforced style for the Kabob Menu for now, and if down the road we need some customization of styling (for Nora or other products) we can refactor or potentially create a new component?

Reply:
No, there's been quite a bit of difficulty in just getting this to work and so I think this was an oversight on my part! I think the best and easiest approach at this point is for me to replace some of the hard coded values with the conditional CSS props pattern that's been getting introduced elsewhere like the data grid and what not e.g. `var(--override-value, var(--fallback-value))`. Thanks for noticing this Steph!

I'm going to just piggyback that in here before merging.

Reply:
Awesome! Thanks for the explanation :)

Reply:
Thank you Steph, nice catch!

I've just pushed up with defs like `--kabob-*` which will achieve ^